all-components/button.css

```diff
@@ -0,0 +1,91 @@
+@import url('/components/utilities.css');
+
+/*common button styling*/
+.button {
+    min-width: 8rem;
+    width: max-content;
+    height: 2.5rem;
+    font-size: medium;
+    cursor: pointer;
+    padding: 0.5rem;
+    border: none;
+}
+
+.round-corner {
+    border-radius: 0.5rem;
+}
+
+
+/*primary button*/
+.primary {
+    background-color: var(--PRIMARY-COLOR);
+    color: var(--WHITE);
+}
+
+.gradient {
+   background: linear-gradient(45deg, var(--PRIMARY-COLOR), var(--SECONDARY-COLOR));
+   color: var(--WHITE);
+}
+
+.secondary {
+    background-color: var(--SECONDARY-COLOR);
+    color: var(--WHITE);
+}
+
+
+/*outline button*/
+.outline-primary {
+    background-color: var(--WHITE);
+    border: 0.15rem solid var(--PRIMARY-COLOR);
+    color: var(--PRIMARY-COLOR);
+}
+
+.outline-primary:hover {
+    background-color: var(--PRIMARY-COLOR);
+    color: var(--WHITE);
+}
+
+.outline-secondary {
+    background-color: var(--WHITE);
+    border: 0.15rem solid var(--SECONDARY-COLOR);
+}
+
+.outline-secondary:hover {
+    background-color: var(--SECONDARY-COLOR);
+    color: var(--WHITE);
+}
+
+
+/*button with icons*/
+.icon {
+    height: 1.25rem;
+    width: 1.25rem;
+}
+
+.button-icons {
+    min-width: max-content;
+    gap: 0.5rem;
```

Comment:
Change this to 5px

Reply:
My root font size is the default 16px, so 0.5rem will be 8px. I have updated gap with 8px.